modules/ocf/manifests/packages.pp

```diff
@@ -125,6 +127,16 @@
     ]:;
   }
 
+  # Packages to only install on Debian (not on Raspbian for example)
+  if $::lsbdistid == 'Debian' {
+    package {
+      [
+        'aactivator',
+        'fluffy',
```

Comment:
I think we can make an apt repo and just upload these (along with ocflib if we can build it correctly) if we'd like to remove this restriction.

Reply:
sounds reasonable to me. may not be worth the effort though.

i'm more interested in ocflib than these tbh

Reply:
Yeah, same on ocflib, I think that's definitely more important.